src/qt/bitcoingui.cpp

```diff
@@ -676,7 +676,7 @@ void BitcoinGUI::addWallet(WalletModel* walletModel)
 {
     if (!walletFrame) return;
 
-    WalletView* wallet_view = new WalletView(platformStyle, walletFrame);
+    WalletView* wallet_view = new WalletView(walletModel, platformStyle, walletFrame);
     if (!walletFrame->addWallet(walletModel, wallet_view)) return;
```

Comment:
Could a commit to simplify like
```diff
-bool WalletFrame::addWallet(WalletModel* walletModel, WalletView* walletView)
+bool WalletFrame::addView(WalletView* walletView)
```

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/398#issuecomment-912071917).